pages/index.tsx

```
@@ -1,22 +1,219 @@
+                    className="todo-checkbox"
+                  />
+                  {editId === todo._id ? (
+                    <>
+                      <input
+                        type="text"
+                        value={editValue}
+                        onChange={handleEditChange}
+                        className="todo-input"
+                      />
+                      <button
+                        onClick={() => handleEditSave(todo._id!)}
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)